### PR TITLE
Remove embedded Anvil

### PR DESCRIPTION
**Pull request by jakelee8**
The embedded Anvil no longer compiles.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @BitskiCo/rust-devs
```
Comment (jakelee8) [inline .github/CODEOWNERS:1]: @jtescher need to give @BitskiCo/rust-devs write access

**Comment by andschneider** [comment]
This is being used. I would prefer to keep it instead of configuring Anvil separately.

**Comment by jakelee8** [comment]
Anvil requires the `git` version of `ethers` and blocks that migration. Anvil also does not build: https://github.com/BitskiCo/bitski-rs/pull/14

**Comment by andschneider** [comment]
> Anvil requires the `git` version of `ethers` and blocks that migration. Anvil also does not build: #14

It works with `0.17.0` instead of git, and builds with a known working commit. Can we just go with that for now?

**Comment by jakelee8** [comment]
Yes

**Comment by andschneider** [comment]
@jakelee8 I changed my mind 🙃. Currently blocking #15 so I'll figure something else out with Anvil

**Review by andschneider — APPROVED** [review]

**Comment by andschneider** [comment]
Just kidding the issue was fixed in Anvil 😂

**Comment by jakelee8** [comment]
😂 well the PR is here if needed

**Pull request closed without merging**